tests/unit/utils/fix-special-char-test.js

```diff
@@ -0,0 +1,22 @@
+import fixSpecialChar from 'dummy/utils/fix-special-char';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | fix special char');
+
+const testCases = [
+    ['a regular string', 'a regular string'],
```

Comment:
Note to self: using the same code over and offer suggests a chance to centralize the logic and avoid duplication. I'll move these fixtures to a shared file after merging.